### PR TITLE
feat(2fa-manager): add 2fa-manager package

### PR DESCRIPTION
**Pull request by ssstelsss**

### CODE REVIEW

**Review by lupenq — APPROVED** [review]

File: packages/2fa-totp-authentication/isomorphic/constants.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const CREATE_SECRET_URL = '/2fa-totp-authentication/create-secret'
```
Comment (pturchik) [inline packages/2fa-totp-authentication/isomorphic/constants.js:1]: /api/2fa-totp-...

**Review by pturchik — CHANGES_REQUESTED** [review]

File: styleguide/server/index.js [hunk 2]
```diff
@@ -43,9 +44,14 @@ startupjsServer({
   })
   const rootPath = options.dirname.replace(/\/styleguide/g, '')
   initUi(ee, { dirname: rootPath })
-  init2fa(ee, { appName: app.name })
   initRecaptcha(ee)
   initRecaptchaDoc(ee)
+  initTwoFAManager(ee, {
+    providers: [{
```
Comment (pturchik) [inline styleguide/server/index.js:50]:
```js
providers: [
  [TotpProvider, options],
  ...
]
```

**Review by pturchik — CHANGES_REQUESTED** [review]

File: packages/2fa-manager/README.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+Add the following lines to `server/index.js`:
+
+```js
+  import { initTwoFAManager } from '@startupjs/2fa-manager/server'
+```
+
+Add function to `startupjsServer`:
+
+```js
+  initTwoFAManager(ee, {
```
Comment (pturchik) [inline packages/2fa-manager/README.md:24]: spaces

File: packages/2fa-manager/client/helpers/getProviders.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
+import { GET_PROVIDERS_URL } from '../../isomorphic'
+
+export default async function getProviders () {
+  try {
+    const { data } = await axios.get(GET_PROVIDERS_URL)
```
Comment (pturchik) [inline packages/2fa-manager/client/helpers/getProviders.js:6]: ?

File: packages/2fa-manager/docs/2fa-manager.en.mdx [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+Add the following lines to `server/index.js`:
+
+```js
+  import { initTwoFAManager } from '@startupjs/2fa-manager/server'
+```
+
+Add function to `startupjsServer`:
+
+```js
+  initTwoFAManager(ee, {
```
Comment (pturchik) [inline packages/2fa-manager/docs/2fa-manager.en.mdx:29]: spaces

File: packages/2fa-manager/docs/2fa-manager.en.mdx [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+You can create your own providers using the `Provider` class from `@startupjs/2fa-manager/Provider`.
+
+Class `Provider(name, send, check)` where:
+
+- `name` - string, provider name.
+- `send()` - function of sending a message with a code.
+- `check(code)` - function for checking the validity of the code.
+  - `code` - the code to check.
+
+## Example
```
Comment (pturchik) [inline packages/2fa-manager/docs/2fa-manager.en.mdx:52]: **Example**

File: packages/2fa-manager/docs/2fa-manager.en.mdx [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+}
+```
+
+Now this `Provider` can be registered with `initTwoFaManager` and used by the name `testProvider`.
+
+## API
+
+### server
+
+### initTwoFaManager(ee, options)
```
Comment (pturchik) [inline packages/2fa-manager/docs/2fa-manager.en.mdx:90]: ?

**Review by pturchik — CHANGES_REQUESTED** [review]

**Pull request merged**